zmq/client.c

```diff
@@ -0,0 +1,37 @@
+//  Hello World client
```

Comment:
Use TABS for indentation. Use [checkpatch](https://docs.kernel.org/dev-tools/checkpatch.html) to validate the source code.